lua/toggleterm/ui.lua

```diff
@@ -273,17 +285,18 @@ end
 --- @param term Terminal
 function M.open_split(size, term)
   local has_open, win_ids = M.find_open_windows()
-  local commands = split_commands[term.direction]
 
   if has_open then
     -- we need to be in the terminal window most recently opened
     -- in order to split it
     local split_win = win_ids[#win_ids]
+    local command = split_command(term.direction, true)
     if config.persist_size then M.save_window_size(term.direction, split_win) end
     api.nvim_set_current_win(split_win)
-    vim.cmd(commands.existing)
+    vim.cmd(command)
   else
-    vim.cmd(commands.new)
+    local command = split_command(term.direction, false)
+    vim.cmd(command)
```

Comment:
There is no need for an interim variable here
```suggestion
    vim.cmd(split_command(term.direction, false))
```